### PR TITLE
<h1> should be first header, followed by <h2> in CSP markdown levels

### PR DESCRIPTION
**Pull request by joshlory**
Remove `blockly/css/common.css` from External (markdown) levels.  As a result, `<h1>` is once again larger than `<h2>` so many levels needed to be updated to have the correct header order.

Removing the Blockly CSS file gets rid of the grey background on markdown levels 🎉.
### Before:

markdown-headers-before
### After:

markdown-headers-after

Unfortunately I'm not sure the best way to code review this change.  I verified the layout of each level after editing it, and the CSP level builders will take a pass over the levels after this change hits level builder.

### CODE REVIEW

**Comment by breville** [comment]
took a scroll through the whole change and didn't see any issues (though that encrypted stuff is tricky :)  lgtm?

**Comment by Hamms** [comment]
What was your process like for updating all these? Are any of these eyes tested?

**Comment by joshlory** [comment]
Hand-edited 😕.  Most will not have Eyes coverage, but I'll keep an eye out for any baselines that need to be updated tomorrow.

File: dashboard/config/scripts/v2_u1l12_student_lesson_introduction.external [hunk 1]
```diff
@@ -5,32 +5,30 @@ css 'unplugged'
...
-# Background
+## Background
 
 It would be impossible for packets to be routed across the internet without IP addresses. The problem with this system is that, while computers are good at referring to other computers by numbers, humans are not. The Domain Name System (DNS) solves this problem so that we can identify a webpage by its name, even when the IP addresses change.
 
-# Vocabulary
+## Vocabulary
 
-<strong>DNS</strong>: an abbreviation for Domain Name System, the Internet's system for converting alphabetic names into numeric IP addresses. 
+* **DNS:** an abbreviation for Domain Name System, the Internet's system for converting alphabetic names into numeric IP addresses.
```
Comment (Hamms) [inline dashboard/config/scripts/v2_u1l12_student_lesson_introduction.external:16]: this a typo?

File: dashboard/config/scripts/v2_u3l03_student_lesson_introduction.external [hunk 2]
```diff
@@ -2,31 +2,34 @@ name 'v2 U3L03 Student Lesson Introduction'
...
 * Optionally try other algorithms
 
-# Resources
+## Resources
 
 * Activity Guide: <a href="https://docs.google.com/document/d/1l2XoaDmn0vYphlT_KS9_Yl7TVgoy6z310qUuUGGKbGg" target="_blank">Human Machine Language - Part 2: Min To Front</a>
 * (Optional) App Lab Widget: (Coming Soon)
 
-
+<a class="btn btn-large btn-primary next-stage submitButton">Continue</a>
```
Comment (Hamms) [inline dashboard/config/scripts/v2_u3l03_student_lesson_introduction.external:32]: why this addition?

**Comment by Hamms** [comment]
That sounds exhausting :P

This LGTM!

File: dashboard/config/scripts/v2_u1l12_student_lesson_introduction.external (hunk 1 above)
Comment (joshlory) [inline dashboard/config/scripts/v2_u1l12_student_lesson_introduction.external:16]: Nope, prefer markdown `**DNS**` over HTML `<strong>DNS</strong>`.  Also moved the colon inside the tag.

File: dashboard/config/scripts/v2_u3l03_student_lesson_introduction.external (hunk 2 above)
Comment (joshlory) [inline dashboard/config/scripts/v2_u3l03_student_lesson_introduction.external:32]: This level was missing an orange "Continue" button.  The button has to be included manually or there's no way to complete the level 😢.

In a future PR I'd like to remove the button from every markdown level and add it to the template.

**Comment by tanyaparker** [comment]
Before/After looks awesome

File: dashboard/config/scripts/v2_u1l12_student_lesson_introduction.external (hunk 1 above)
Comment (Hamms) [inline dashboard/config/scripts/v2_u1l12_student_lesson_introduction.external:16]: Should have been more specific; I was referring to the added bullet point

Comment (joshlory) [inline dashboard/config/scripts/v2_u1l12_student_lesson_introduction.external:16]: Ah, yes, also intentional.

**Pull request merged**